tests/test_orbit.py

```diff
@@ -582,6 +584,7 @@ def test_energy_symplec_longterm():
     #Basic parameters for the test
     times= numpy.linspace(0.,10000.,100001) #~360 Gyr at the Solar circle
     integrators= ['leapfrog_c', #don't do leapfrog, because it takes too long
+                  'dop853',
```

Comment:
should `dop853` be added here? it passes the test but it is not symplectic which is this test is about???

Reply:
Good that it passes the test! But if it's not symplectic, there should be some long-term energy error growth (do ``o.plotE()``), so it shouldn't be in this test (which is for symplectic integrators).